api/src/main/java/org/openmrs/logging/OpenmrsConfigurationFactory.java

```diff
@@ -103,6 +114,33 @@ protected String[] getSupportedTypes() {
 		return SUFFIXES;
 	}
 	
+	public List<File> getConfigurationFiles() {
```

Comment:
The idea of this method is to find anything in the .OpenMRS directory or .OpenMRS/configuration directory (non-recursively) that is named log4j2*.{extension}.

It returns these sorted by name (without the directory), so inheritance will be based solely on filename sorting across these 2 directories.

Not sure if this is the best way to determine inheritance / which override which, but this is the most straightforward.